commcare_connect/opportunity/tests/test_management_commands.py

```
@@ -1,9 +1,153 @@
+
+        call_command("fix_managed_opp_approved_counts", "--no-dry-run")
+
+        cw.refresh_from_db()
+        assert cw.saved_approved_count == 1
+        assert cw.saved_payment_accrued == 100
+        # The access-level total (which feeds invoicing) must be recomputed too.
+        opp_access.refresh_from_db()
+        assert opp_access.payment_accrued == 100
+
+    def test_rerun_is_idempotent(self):
+        opp_access, cw = self._setup_managed_opp_with_stale_count()
```

Ruff (0.15.18)

[warning] Unpacked variable `opp_access` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)